tests/testthat/test-geoknife_output_parser.R

```diff
@@ -55,4 +55,9 @@ test_that("timeseries parser works on multi var multi feature", {
   expect_error(parseTimeseries(local_file, delim = ' '))
 })
 
+test_that("timeseries parser works on multi threshold output",{
+  local_file <- system.file('extdata','csv_multi_threshold.csv', package = 'geoknife')
+  output <- parseTimeseries(local_file, delim = ',')
+  expect_is(output, "data.frame")
```

Comment:
I'm not sure it is exactly correct to say it is test driven when the only expect is that the type of the output is a data.frame.